### PR TITLE
Rules for Memento Mori + RedMountainsReborn + BCOM

### PR DESCRIPTION
**Pull request by ActuallyUlysses**
This is rule for a patch for MM + RMR + BCOM. I published in on discord and its currently being tested for left over seams, ters and floaters - Link to discord https://discord.com/channels/210394599246659585/847622100218544158/1097228510579413053

RedMountainReborn.esp must be the one supplied with the patch or wild things will happen

### CODE REVIEW

**Comment by rfuzzo** [comment]
hmm, can you post a quick message when it's done being tested? I can merge then, also probably should be posted to nexus (and not just a link to github?)

**Review by rfuzzo — CHANGES_REQUESTED** [review]
hmm, can you post a quick message when it's done being tested? I can merge then, also probably should be posted to nexus (and not just a link to github?)

**Comment by ActuallyUlysses** [comment]
Yup. Also Amelie just realized that I made a mistake.

There is no need to put MementoMori.esp in the rule. RMR replacer got specific cells removed and patch just fixes land tears and seams.

File: mlox_user.txt [hunk 1]
```diff
@@ -4165,6 +4165,11 @@ Memento Mori.esp
 Memento Mori.esp
 Memento Mori - OpenMW Patch.esp
 
+[Order] ; (ActuallyUlysses) This is a patch for MM + RMR + BCOM. I published in on discord and its currently being tested for left over seams, ters and floaters - Link to discord https://discord.com/channels/210394599246659585/847622100218544158/1097228510579413053
+Memento Mori.esp
```
Comment (ActuallyUlysses) [inline mlox_user.txt:4169]: Remove that. No need and might cause issues with other rules.

**Review by rfuzzo — APPROVED** [review]
lgtm

**Pull request merged**